src/main/java/com/exasol/adapter/request/renderer/PushdownSqlRenderer.java

```diff
@@ -65,6 +65,19 @@ private void addIfPresent(final SqlNode node, final String key, final JsonObject
             }
         }
 
+        private void addGroupByIfPresent(final SqlExpressionList groupBy, final JsonObjectBuilder builder)
+                throws AdapterException {
+            if (groupBy != null) {
+                final JsonValue groupByJson = groupBy.accept(this);
+                if (groupByJson.toString().contains("single_group")) {
```

Comment:
that looks dangerous. The json could also contain `single_group` as a column name.